### ISSUE TITLE
Production Worker - Mass Material Movement

### ISSUE THREAD

**Issue by aellis** [bezlio/bezlio-gallery#47]
Port our Epicor Mass Material Movement customization to Bezlio.  We have existing customers that will get this as a part of maintenance on that legacy product and also use this for new inbounds.

**Comment by gramang** [comment]
@aellis Can you please give us more information on this to understand the requirement

**Comment by aellis** [comment]
@gramang this one will require us getting you up to speed on development within Bezlio itself before you'd be able to work on this one.  Would you have some time Thursday morning for a training session on that?

**Comment by gramang** [comment]
@aellis Yes we are available for the training on Thursday Morning, please let us know the time
